lib/pem/manager.rb

```diff
@@ -1,30 +1,76 @@
+require 'pathname'
+require 'spaceship'
+
 module PEM
   # Creates the push profile and stores it in the correct location
   class Manager
+
     def self.start
-      path, rsa_file = PEM::CertManager.new.run
-
-      if path
-        file_name = PEM.config[:pem_name] || File.basename(path)
-        file_name.concat('.pem') unless file_name =~ /.pem/
-        output = "./#{file_name}"
-        FileUtils.mv(path, output)
-        path = output
-        puts output.green
-      end
-      
-      if path and rsa_file
-        if PEM.config[:save_private_key]
-          file_name = File.basename(rsa_file)
-          output = "./#{file_name}"
-          FileUtils.mv(rsa_file, output)
-          puts output.green          
+      begin
+        password_manager = CredentialsManager::PasswordManager.shared_manager
+        Spaceship.login(password_manager.username, password_manager.password)
+        if Spaceship.client.teams.size > 1
```

Comment:
Are you sure that's necessary? I think it's better solved in `sigh` using environment variables afaik.